src/main/java/com/firebase/geofire/GeoQuery.java

```diff
@@ -112,8 +115,8 @@ private boolean locationIsInQuery(GeoLocation location) {
     }
 
     private void postEvent(Runnable r) {
-        EventTarget target = Firebase.getDefaultConfig().getEventTarget();
-        target.postEvent(r);
+        Repo repo = geoFire.getDatabaseReference().getRepo();
```

Comment:
I could not find a way to access the EventTarget similar to the existing code. After decompiling the library and looking around, this looked like the closest replication of the existing behavior. If you  have a recommendation regarding how I should have done this I can update the PR.